README.adoc

```diff
@@ -285,6 +285,102 @@ spec:
 ...
 ----
 
+== Configure Health
+
+=== Kiali defaults
+
+By default Kiali health is:
+
+* Degraded when Error Rate is >= 0.1%
+* Failure when Error Rate is >= 20%
+
+For Kiali an error request is when:
+
+* For http the code status match the expression */^[4-5]\d\d$/*
+* For grpc the code status match the expression */^ [1-9]$|^1[0-6]$/*
+
+=== Configuration
+
+Kiali works with a FIFO configuration, so Kiali will use the first configuration that applies with the resource requested.
+
+So the granularity and criteria should be configured from highest to lowest.
```

Comment:
You assume that user knows about granularity and criteria and that's not described on the feature.